Cargo.toml

```diff
@@ -188,3 +189,5 @@ authority-selection-inherents = { path = "primitives/authority-selection-inheren
 session-manager = { path = "primitives/session-manager", default-features = false }
 sp-sidechain = { path = "primitives/sidechain", default-features = false }
 chain-params = { path = "primitives/chain-params", default-features = false }
+pallet-native-token-management = { path = "pallets/native-token-management", default-features = false }
```

Comment:
Design mentioned in the PR description has two pallets: "native token transfer pallet" and "native token management pallet". To be honest I think they are meant to be so related that I think having one pallet is better but still this is discrepant to design = mess.

Reply:
Yes, they are extremely similar and I need to make sure we really need to distinguish between those cases (release from reserve vs. someone just sending tokens to the illiquid supply - they all end up in the same place)

Reply:
I've gone through the requirements document and concluded there's nothing distinguishing the two types of event from the observability perspective (they only differ on the MC). I've modified the design in Miro to use a single pallet like this PR does.